api/envoy/extensions/dso/v3/dso.proto

```diff
@@ -0,0 +1,150 @@
+syntax = "proto3";
```

Comment:
All of this should be in contrib. Since it's going in contrib, for now I would just merge the DSO functionality directly into the Go filter. We can move it out later if needed.

Reply:
OK. I have merged the DSO functionality into the Go filter.